packages/espressocash_api/lib/src/dto/scalex.dart

```diff
@@ -22,6 +22,8 @@ class GenerateScalexLinkRequestDto with _$GenerateScalexLinkRequestDto {
     required String type,
     required String address,
     required String email,
+    required double amount,
+    required String currency,
```

Comment:
It cannot become required, it will break current clients if backend starts requiring it.